### ISSUE TITLE
Fix no-undef and no-unused-vars eslint violations

### ISSUE THREAD

**Issue by cdoremus** [5calls/5calls#238]
We have a number of ```no-undef``` and ```no-unused-vars``` eslint rule violations that need to be fixed. They are listed below with ```main.js``` containing 84 of the 117 violations. 

These violations are configured as warnings, but the eslint recommendation is to make them an error. Our warning designation was done to prevent CircleCI failures.

Once these errors have been fixed,  the ```no-undef``` and ```no-unused-vars``` rules should be set to "error" in the root ```.eslintrc.json``` file.

```
/home/craig/git/5calls/static/js/components/about.js
  5:54  warning  'e' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/call_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/call.js
  14:49  warning  'e' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/callcount_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/callcount.js
  5:35  warning  'e' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/contact_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/contact.js
   2:7   warning  'find' is assigned a value but never used  no-unused-vars
  11:3   warning  'repID' is not defined                     no-undef
  13:5   warning  'repID' is not defined                     no-undef
  47:52  warning  'repID' is not defined                     no-undef

/home/craig/git/5calls/static/js/components/content.js
  19:86  warning  'e' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/done_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/done.js
  11:54  warning  'e' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/hypothesis.js
  7:39  warning  'e' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/issuesList_test.js
   1:7   warning  'html' is assigned a value but never used  no-unused-vars
  23:23  warning  'objWithId' is defined but never used      no-unused-vars

/home/craig/git/5calls/static/js/components/issuesListItem_test.js
   1:7   warning  'html' is assigned a value but never used        no-unused-vars
  16:9   warning  'sendCalled' is assigned a value but never used  no-unused-vars
  17:23  warning  'objWithId' is defined but never used            no-unused-vars
  39:9   warning  'sendCalled' is assigned a value but never used  no-unused-vars
  40:23  warning  'objWithId' is defined but never used            no-unused-vars

/home/craig/git/5calls/static/js/components/issuesListItem.js
  20:24  warning  'e' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/issuesLocation_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/issuesLocation.js
  42:5   warning  'address' is not defined  no-undef
  44:25  warning  'address' is not defined  no-undef

/home/craig/git/5calls/static/js/components/noContact_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/noContact.js
  2:7  warning  'find' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/outcomes_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/promote_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/script_test.js
  1:7  warning  'html' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/components/scriptLine.js
  3:44  warning  'send' is defined but never used  no-unused-vars

/home/craig/git/5calls/static/js/main.js
    2:7    warning  'html' is assigned a value but never used  no-unused-vars
   23:1    warning  'cachedAddress' is not defined             no-undef
   26:4    warning  'cachedAddress' is not defined             no-undef
   31:1    warning  'cachedGeo' is not defined                 no-undef
   35:5    warning  'cachedGeo' is not defined                 no-undef
   39:31   warning  'cachedGeo' is not defined                 no-undef
   42:1    warning  'cachedAllowBrowserGeo' is not defined     no-undef
   46:5    warning  'cachedAllowBrowserGeo' is not defined     no-undef
   50:32   warning  'cachedAllowBrowserGeo' is not defined     no-undef
   53:1    warning  'cachedGeoTime' is not defined             no-undef
   57:5    warning  'cachedGeoTime' is not defined             no-undef
   61:1    warning  'cachedCity' is not defined                no-undef
   65:5    warning  'cachedCity' is not defined                no-undef
   69:28   warning  'cachedCity' is not defined                no-undef
   70:28   warning  'cachedAddress' is not defined             no-undef
   73:1    warning  'completedIssues' is not defined           no-undef
   75:3    warning  'completedIssues' is not defined           no-undef
   86:14   warning  'cachedAddress' is not defined             no-undef
   89:18   warning  'cachedGeo' is not defined                 no-undef
   90:19   warning  'cachedGeoTime' is not defined             no-undef
   91:22   warning  'cachedAllowBrowserGeo' is not defined     no-undef
   92:17   warning  'cachedCity' is not defined                no-undef
  102:22   warning  'completedIssues' is not defined           no-undef
  111:7    warning  'response' is not defined                  no-undef
  112:7    warning  'issues' is not defined                    no-undef
  112:16   warning  'response' is not defined                  no-undef
  113:7    warning  'contactIndices' is not defined            no-undef
  114:7    warning  'issues' is not defined                    no-undef
  114:37   warning  'index' is defined but never used          no-unused-vars
  115:10   warning  'contactIndices' is not defined            no-undef
  117:24   warning  'issues' is not defined                    no-undef
  117:47   warning  'response' is not defined                  no-undef
  117:87   warning  'response' is not defined                  no-undef
  117:128  warning  'contactIndices' is not defined            no-undef
  120:7    warning  'totals' is not defined                    no-undef
  121:28   warning  'totals' is not defined                    no-undef
  124:7    warning  'geo' is not defined                       no-undef
  125:7    warning  'city' is not defined                      no-undef
  126:7    warning  'time' is not defined                      no-undef
  127:50   warning  'geo' is not defined                       no-undef
  128:55   warning  'city' is not defined                      no-undef
  129:55   warning  'time' is not defined                      no-undef
  130:29   warning  'geo' is not defined                       no-undef
  130:46   warning  'city' is not defined                      no-undef
  130:66   warning  'time' is not defined                      no-undef
  133:7    warning  'contactIndices' is not defined            no-undef
  135:9    warning  'contactIndices' is not defined            no-undef
  136:34   warning  'contactIndices' is not defined            no-undef
  138:9    warning  'contactIndices' is not defined            no-undef
  139:34   warning  'contactIndices' is not defined            no-undef
  143:7    warning  'Raven' is not defined                     no-undef
  153:7    warning  'response' is not defined                  no-undef
  154:11   warning  'response' is not defined                  no-undef
  155:57   warning  'response' is not defined                  no-undef
  156:30   warning  'response' is not defined                  no-undef
  168:28   warning  'data' is defined but never used           no-unused-vars
  175:28   warning  'data' is defined but never used           no-unused-vars
  182:35   warning  'data' is defined but never used           no-unused-vars
  186:19   warning  'data' is defined but never used           no-unused-vars
  189:39   warning  'data' is defined but never used           no-unused-vars
  190:37   warning  'data' is defined but never used           no-unused-vars
  195:7    warning  'address' is not defined                   no-undef
  197:9    warning  'address' is not defined                   no-undef
  199:9    warning  'address' is not defined                   no-undef
  202:42   warning  'address' is not defined                   no-undef
  234:13   warning  'response' is not defined                  no-undef
  235:17   warning  'response' is not defined                  no-undef
  236:40   warning  'response' is not defined                  no-undef
  240:15   warning  'Raven' is not defined                     no-undef
  240:62   warning  'response' is not defined                  no-undef
  244:13   warning  'Raven' is not defined                     no-undef
  245:13   warning  'Raven' is not defined                     no-undef
  250:11   warning  'Raven' is not defined                     no-undef
  303:40   warning  'done' is defined but never used           no-unused-vars
  328:34   warning  'done' is defined but never used           no-unused-vars
  329:7    warning  'ga' is not defined                        no-undef
  334:7    warning  'currentIndex' is not defined              no-undef
  335:11   warning  'currentIndex' is not defined              no-undef
  337:47   warning  'currentIndex' is not defined              no-undef
  349:9    warning  'ga' is not defined                        no-undef
  351:9    warning  'ga' is not defined                        no-undef
  355:126  warning  'body' is defined but never used           no-unused-vars
  363:7    warning  'ga' is not defined                        no-undef
  370:7    warning  'ga' is not defined                        no-undef

/home/craig/git/5calls/static/js/pages/mainView.js
  2:7  warning  'store' is assigned a value but never used  no-unused-vars

/home/craig/git/5calls/static/js/utils/localstorage.js
  23:39  warning  'items' is defined but never used  no-unused-vars

✖ 117 problems (0 errors, 117 warnings)
```

**Comment by cdoremus** [comment]
This depends on #229 being merged into master

**Comment by cdoremus** [comment]
Merging of PR #236 eliminates all the ```main.js``` violations. Thanks, Greg (@greggraf)!

**Comment by greggraf** [comment]
Created PR #244 to handle the rest of the files.

**Comment by cdoremus** [comment]
Remaining issues fixed in issue #244 that has been merged into master.
